### PR TITLE
[ ADMIN ] Add code of conduct

### PR DESCRIPTION
**Pull request by jijoongmoon**
Add documentaions releate with code of conduct.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: docs/coding-convention.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+$ ./tools/development/indent.sh <file-name>
+```
+
+## C headers (.h)
+
+You may indent differently from what indent.sh does. You may also break the 80-column rule with header files.
+
+Except the two, you are required to follow the general coding styles mandated by gst-indent
+
+## C++ files (.cc)
```
Comment (zhoonit) [inline docs/coding-convention.md:23]:
```suggestion
## C++ files (.cpp)
```

**Review by zhoonit — APPROVED** [review]
LGTM except minor comment.

Currently we are using clang format for `.c` and `.h` (for checking with ci as well). Are we gonna use gnuindent sooner or later?

File: docs/coding-convention.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Other files
+
+- [Java] TBD
+- [Python] TBD
+- [Bash] TBD
+
+
+# File Locations
+
+## Directory structure of nnstreamer.git
```
Comment (kparichay) [inline docs/coding-convention.md:38]: nntrainer.git?

File: docs/coding-convention.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- **api**: API definitions and implementations
+    - **capi**: C-APIs (Tizen and others)
```
Comment (kparichay) [inline docs/coding-convention.md:41]: I think we can ccapi here? or later maybe.

File: docs/coding-convention.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- **nnstreamer**: NNStreamer sub-filter codes for NNTrainer
+- **jni**: Android/Java build scripts.
+- **Applications**: Examples for NNtrainer
+- **packaging**: Tizen RPM build scripts. OpenSUSE/Redhat Linux may reuse this.
+- **test**: Unit test cases. We have GTEST test cases. There are subdirectories, which are groups of unit test cases.
+- **tools**: Various developmental tools and scripts of NNTrainer.
+
+## Related git repositories
+
+- [NNStreamer](https://github.com/nnstreamer/nnstreamer)
```
Comment (kparichay) [inline docs/coding-convention.md:54]: also add https://github.com/nnstreamer/api
this will soon become the API repo.

**Review by kparichay — APPROVED** [review]
Overall LGTM

File: docs/coding-convention.md (hunk 3 above)
Comment (jijoongmoon) [reply docs/coding-convention.md:41]: I think later.

File: docs/coding-convention.md (hunk 4 above)
Comment (jijoongmoon) [reply docs/coding-convention.md:54]: I think later.

File: docs/coding-convention.md (hunk 2 above)
Comment (jijoongmoon) [reply docs/coding-convention.md]: fixed. thanks.

File: docs/coding-convention.md (hunk 1 above)
Comment (jijoongmoon) [reply docs/coding-convention.md]: fixed. thanks.

**Comment by jijoongmoon** [comment]
> LGTM except minor comment.
> 
> Currently we are using clang format for `.c` and `.h` (for checking with ci as well). Are we gonna use gnuindent sooner or later?

Yes. You are right. We will use clang-format only.

**Pull request merged**